lib/krane/cli/deploy_command.rb

```diff
@@ -16,6 +16,8 @@ class DeployCommand
         "filenames" => { type: :array, banner: 'config/deploy/production config/deploy/my-extra-resource.yml',
                          aliases: :f, required: true,
```

Comment:
This is no longer required if we can accept `STDIN`. However, we have to ensure that it is not empty after parsing options

Reply:
an empty check `options[:filenames].empty?` is given, we'll raise Thor RequiredArgumentMissingError if it's empty